### DISCUSSION TITLE
When I use scene viewer the model is placed sideways.

### DISCUSSION THREAD

**Discussion by F1shez** [google/model-viewer#4730] [Q&A]
### Description
Hello all, I'm trying to display the model in ar with standart scene viewer and the model is placed sideways.

#### Live Demo
Example: https://modelviewer.dev/examples/augmentedreality/#sceneViewer

### Version
- model-viewer: v3.4.0
- Google Play Services for AR 1.42.240360593

### Browser Affected
- [x] Chrome, version: 123.0.6312.80
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS
- [x] Android 14
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [x] SceneViewer
- [ ] QuickLook

![photo_2024-03-29_15-20-18](https://github.com/google/model-viewer/assets/22225036/fb2e9df4-d677-4d72-9df4-16b523bae98b)

**Comment by F1shez** [comment]
perhaps this can be influenced by disable_occlusion=true, because if you try on a tiger from the official Google website, then everything is fine [link to open tiger](https://www.google.com/search?q=tiger&oq=tiger&gs_lcrp=EgZjaHJvbWUqDggAEEUYJxg7GIAEGIoFMg4IABBFGCcYOxiABBiKBTIMCAEQABhDGIAEGIoFMg8IAhAAGEMYsQMYgAQYigUyDAgDEAAYQxiABBiKBTIMCAQQABhDGIAEGIoFMhAIBRAuGMcBGLEDGNEDGIAEMgwIBhAAGEMYgAQYigUyDAgHEAAYQxiABBiKBTIMCAgQABgUGIcCGIAEMgwICRAAGEMYgAQYigUyFAgKEC4YChhDGMcBGNEDGIAEGIoFMgoICxAAGLEDGIAEMgwIDBAAGEMYgAQYigUyDwgNEAAYQxixAxiABBiKBTIMCA4QABhDGIAEGIoF0gEHOTM5ajBqNKgCDrACAQ&client=ms-android-google&sourceid=chrome-mobile&ie=UTF-8#)

**Reply by elalish** [reply]
In what way is the provided image sideways? Can you provide an image of what you expect the behavior to be for comparison? A video might also help so we can see how you got to this result.

**Reply by F1shez** [reply]
sorry, i now have problem and with tiger, object placed random rotated, link for video demonstration on youtube: https://www.youtube.com/watch?v=MO59vjPHgvg.

**Reply by elalish** [reply]
Is this a duplicate of #4732?

**Reply by F1shez** [reply]
no, in two links in the post #4732 when calling scene viewer, i have random rotate placed object. Example object placed on the left side.
![Screenshot_20240404-203157](https://github.com/google/model-viewer/assets/22225036/af6d0ab8-b5cd-4e68-938e-205fc4218f2c)
